lib/Service/ColumnService.php

```diff
@@ -362,48 +362,30 @@ public function update(
 			if ($subtype !== null) {
 				$item->setSubtype($subtype);
 			}
-			if ($numberPrefix !== null) {
+			if($numberPrefix !== null) {
```

Comment:
```suggestion
			if ($numberPrefix !== null) {
```

Not sure why php-cs-fixer doesn't complain or auto fix this, but would be more aligned to the rest of the code base